src/data/challenge.ts

```diff
@@ -666,6 +680,41 @@ export class FreshStartChallenge extends Challenge {
   }
 }
 
+/**
+ * Implements an inverse battle challenge.
+ */
+export class InverseBattleChallenge extends Challenge {
+  constructor() {
+    super(Challenges.INVERSE_BATTLE, 1);
+  }
+  /**
+   * @overrides
+   */
+  getDifficulty(): number {
```

Comment:
NIT: you can use `override` as a keyword

```suggestion
  override getDifficulty(): number {
```

Reply:
i added override as a keyword and removed comments